### ISSUE TITLE
File names on libraries text or libraries display

### ISSUE THREAD

**Issue by DarwinWasWrong** [wokwi/wokwi-features#403]
When adding libraries, it is sometimes a frustrating search the for correct file name for the include statements.
A very simple example of this is
LCD IC2 library requires
 #include <LiquidCrystal_I2C.h>
 
libraries.txt shows LiquidCrystal I2C
![image](https://user-images.githubusercontent.com/6262140/184759612-23526ab1-ea00-42c1-9f8c-c42f0472bb4f.png)

Library Manager Shows
![image](https://user-images.githubusercontent.com/6262140/184759555-b71c6a6b-0a19-427a-bcf9-f0ebc0fd1d4f.png)

This is the simplest example
Try DHT
![image](https://user-images.githubusercontent.com/6262140/184759735-1322838c-01b3-4957-a3e6-8c0cdd57858c.png)

If the library text OR the installed libraries page has the actual library file name included, this would help a lot.

**Comment by urish** [comment]
Good idea! Opened [for voting](https://wokwi.com/features?sort=newest#feature-1339676240)

**Comment by Twistx77** [comment]
Hi, wouldn't it ever to just use something like the VSCode Integration feature help with this? You could just use Platformio IO to manage the libraries.
The VSCode integration is already working in alfa. Maybe it makes more sense to vote for it rather than this? https://github.com/wokwi/wokwi-features/issues/414

**Comment by DarwinWasWrong** [comment]
No, it is a simple change to the UI to allow beginners etc to find what is required to run a library
I don't use VS code

**Comment by urish** [comment]
For popular libraries, Wokwi now automatically suggests the library based on your includes, e.g.:

<img width="1612" height="627" alt="Image" src="https://github.com/user-attachments/assets/3ec94ae8-1b06-4ffd-9eea-4d6d4463aa65" />
